lib/internal/errors.js

```diff
@@ -192,11 +193,25 @@ E('ERR_V8BREAKITERATOR', 'full ICU data not installed. ' +
 function invalidArgType(name, expected, actual) {
   const assert = lazyAssert();
   assert(name, 'name is required');
-  var msg = `The "${name}" argument must be ${oneOf(expected, 'type')}`;
-  if (arguments.length >= 3) {
-    msg += `. Received type ${actual !== null ? typeof actual : 'null'}`;
+  assert.strictEqual(arguments.length, 3, 'arguments length mismatch');
+  // Only check the first entry
+  const expectedType = Array.isArray(expected) ? expected[0] : expected;
+  // A-Z = 65-90, a-z = 97-122
+  const type = expectedType.charCodeAt(0) > 96 ? 'of type' : 'instance of';
+  var received;
+  if (actual == null) {
+    received = String(actual);
+  } else if (typeof actual === 'object') {
+    if (actual.constructor !== undefined) {
```

Comment:
Hm, as far as I can tell someone would have to explicitly set the constructor to null... that would be really weird but I'll change it accordingly.